README.md

```diff
@@ -900,14 +900,17 @@ Each item in the array is an object with the following properties:
 
 **Example HTML output:**
 
-    <h1 id="hello-world">Hello World!</h1>
+    <h1 id="hello-world"><a class="hash-anchor" href="#hello-world" aria-hidden="true"></a>Hello World!</h1>
 
 **Options:**
 
 Pass options for this plugin as the `heading_id` property of the `do-markdownit` plugin options.
 Set this property to `false` to disable this plugin.
 
 - `sluggify` (`function(string): string`, optional): Custom function to convert heading content to a slug Id.
+- `hashLink` Set this property to `false` to disable this feature.
+    - `maxLevel` (`number`, optional, defaults to `3`): Max number of heading levels to generate a hash link.
```

Comment:
```suggestion
    - `maxLevel` (`number`, optional, defaults to `3`): Max heading level to generate hash links for.
```
Nit pick on wording, I felt `Max number of heading levels` could be misread as "this will only add 3 headings"